examples/plugins/inmemkv/inmemkv_backend.cpp

```
@@ -0,0 +1,455 @@
+    nixlInMemKVBackendReqH() = default;
+    ~nixlInMemKVBackendReqH() = default;
+};
+
+/**
+ * @brief Metadata for INMEMKV memory descriptors
+ *
+ * Stores the mapping between devId and key for later lookups.
+ */
+class nixlInMemKVMetadata : public nixlBackendMD {
+public:
+    nixlInMemKVMetadata(uint64_t dev_id, std::string key)
```

Cppcheck (2.20.0)

[performance] Function parameter 'key' should be passed by const reference.

(passedByValue)